fallaxy.py

```diff
@@ -743,37 +721,27 @@ def collections_post(api_version):
         return json_galaxy_error(api_version, 'unsupported media type',
                                  'Unsupported medial type "%s" in request.' % content_type_header, status=415)
 
-    boundary = re.findall(r'boundary=([\x00-\x7f]*)', boundary)[0]  # Boundary can take in any 7-bit ASCII chars.
-    if not boundary:
-        return json_galaxy_error(api_version, 'parse_error',
-                                 'Multipart form parse error - Invalid boundary in multipart: ')
```

Comment:
I didn't restore this check, as I'm not sure it's useful. The underlying `werkzeug` code should be validating this, but since this is flask, and galaxy is django, I guess there could be inconsistency, but not sure it's important for our use.